test/test_named_destinations.py

```diff
@@ -0,0 +1,78 @@
+"""Unit tests for named destinations in links for FPDF2."""
+
+import os
+import pytest
+from fpdf import FPDF
+
```

Comment:
```suggestion
from test.conftest import assert_pdf_equal

HERE = Path(__file__).resolve().parent
```

You'll need those imports